app/footer.tsx

```diff
@@ -1,3 +1,182 @@
-export default function Footer() {
-  return <></>;
+import Image from 'next/image';
+import footerimg from '../public/assets/footerimg.png';
```

Comment:
Get these images from Figma instead of keeping it in the Git tree. See #45 for reference

Reply:
Okay so i should remove the public/assets folder ? And you want me to add the images through the urls from figma?

Reply:
Yes.